web/src/components/gnd-legend/index.js

```diff
@@ -174,7 +174,7 @@ class GndLegend extends React.Component<Props> {
       <Card className={classes.card}>
         <CardContent className={classes.cardContent}>
           <Typography className={classes.heading} color="textSecondary">
-            Legend
+            Layer
```

Comment:
Please use "Layers" instead (plural), since the panel currently shows all layers available, not just one.

Reply:
Thanks! Updated the code.